classes/UpgradePage.php

```diff
@@ -177,7 +177,7 @@ public function display($ajaxResult)
         $errMessageData = $this->getErrorMessage();
         if (!empty($errMessageData)) {
             return $twig
-                ->render('@ModuleAutoUpgrade:error.twig', $errMessageData);
+                ->render('@ModuleAutoUpgrade:error.html.twig', $errMessageData);
```

Comment:
```suggestion
                ->render('@ModuleAutoUpgrade/error.html.twig', $errMessageData);
```

Reply:
Are you sure it's working with all Symfony versions?

Reply:
Yep: [https://symfony.com/doc/2.8/templating/namespaced_paths.html](https://symfony.com/doc/2.8/templating/namespaced_paths.html)